file-formats/cdbo/cdbo/type/zif_aff_cdbo_v1.intf.abap

```diff
@@ -0,0 +1,155 @@
+
+  CONSTANTS:
+    "! <p class="shorttext">CDS View Type</p>
+    "! CDS View type
+    BEGIN OF co_cds_type,
+      "! <p class="shorttext">Basic</p>
+      "! Basic
+      basic       TYPE ty_cds_view_type VALUE 'BASIC',
+      "! <p class="shorttext">Composite</p>
+      "! Composite
+      composite   TYPE ty_cds_view_type VALUE 'COMPOSITE',
+      "! <p class="shorttext">Consumption</p>
+      "! Consumption
+      consumption TYPE ty_cds_view_type VALUE 'CONSUMPTION',
+      "! <p class="shorttext">Extension</p>
+      "! Extension
+      extension   TYPE ty_cds_view_type VALUE 'EXTENSION',
+    END OF co_cds_type.
+
+  "! <p class="shorttext">CDS View Status</p>
+  "! CDS view status
+  "! $values {@link zif_aff_cdbo_v1.data:co_cds_view_status}
+  "! $default {@link zif_aff_cdbo_v1.data:co_cds_view_status.released}
+  TYPES ty_cds_view_status TYPE c LENGTH 60.
+
+  CONSTANTS:
+    "! <p class="shorttext">CDS View Status</p>
+    "! CDS View type
+    BEGIN OF co_cds_view_status,
+      "! <p class="shorttext">Released</p>
+      "! Released
+      released TYPE ty_cds_view_status VALUE 'RELEASED',
+      "! <p class="shorttext">Not Released</p>
+      "! Not Realeased
+      not_released    TYPE ty_cds_view_status VALUE ' ',
+    END OF co_cds_view_status.
+
+  "! <p class="shorttext">Field Nature</p>
+  "! Nature of the field
+  "! $values {@link zif_aff_cdbo_v1.data:co_field_nature}
+  TYPES ty_field_nature TYPE c LENGTH 2.
+
+  CONSTANTS:
+    "! <p class="shorttext">Field Nature</p>
+    "! Nature of the field
+    BEGIN OF co_field_nature,
+      "! <p class="shorttext">General fields that do not contain sensitive personal info.</p>
+      "! General fields that do not contain sensitive personal info.
+      general TYPE ty_field_nature VALUE 'GE',
+      "! <p class="shorttext">Sensitive personal information fields including key fields</p>
+      "! Sensitive personal information fields including key fields
+      sensitive    TYPE ty_field_nature VALUE 'SP',
+      "! <p class="shorttext">Non Business Information</p>
+      "! Non business information
+      non_business_information TYPE ty_field_nature VALUE 'NB',
+    END OF co_field_nature.
+
+
+  TYPES:
+    "! <p class="shorttext">CDBO Details</p>
+    "! CDBO deatils
+    BEGIN OF ty_object_details,
+      "! <p class="shorttext">Business Area</p>
+      "! Business area to which the object belongs
+      "! $required
+      business_area         TYPE  c LENGTH 60,
+      "! <p class="shorttext">Sub Business Area</p>
+      "! Sub Business area to which the object belongs
+      "! $required
+      sub_business_area     TYPE  c LENGTH 60,
+      "! <p class="shorttext">Application Component</p>
+      "! Application component to which object belongs
+      "! $required
+      application_component TYPE c LENGTH 24,
+      "! <p class="shorttext">CDS View Type</p>
+      "! Type of the CDS View
+      cds_view_type   TYPE ty_cds_view_type,
```

Comment:
I guess, also the CDS view type is not part of the actual model of CDBO objects, is it?

Reply:
CDS Type is now not a part of AFF